File: 17-单例OC/17-单例OC/ViewController.m
//
//  ViewController.m
//  17-单例OC
//
//  Created by Wei Zhou on 03/10/2016.
//  Copyright © 2016 Wei Zhou. All rights reserved.
//

#import "ViewController.h"
#import "NetworkTools.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    NSLog(@"\nHello, Chacha! I'm a singleton");
    NSLog(@"\n%@", [NetworkTools shareNetworkTools]);
}


@end

File: 16-单例/16-单例/ViewController.swift
//
//  ViewController.swift
//  16-单例
//
//  Created by Wei Zhou on 03/10/2016.
//  Copyright © 2016 Wei Zhou. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        print("这是一个单例", NetworkTools.self, NetworkTools.shareInstance)
    }

}

